docs/reference/buildx_bake.md

```diff
@@ -18,6 +18,7 @@ Build from a file
 | [`--builder`](#builder)             | `string`      |         | Override the configured builder instance                                                            |
 | [`--call`](#call)                   | `string`      | `build` | Set method for evaluating build (`check`, `outline`, `targets`)                                     |
 | [`--check`](#check)                 | `bool`        |         | Shorthand for `--call=check`                                                                        |
+| `-D`, `--debug`                     | `bool`        |         | Enable debug logging                                                                                |
```

Comment:
Do we want both the `-D` and the `--debug` to be inherited, or should we limit to only the long form (`--debug`) for now? (can we?)

cc @jalonsogo

Reply:
I think it should be invisible to user if this flag is defined in the cli or buildx binary. `docker --debug buildx` still uses the defined in CLI.

Reply:
Mostly looking here that the `-D` is documented for the top-level, but for the subcommands it's a new flag, and in most (if not all) cases we refer to it as `--debug`.

Where possibly I try to avoid shorthand flags if they're not _critical_ to have (it's easy to add them later if there's a strong need), more so if they're case-sensitive (if there's both a `-d` and ` -D` flag, which can be confusing).

Reply:
>  but for the subcommands it's a new flag, and in most (if not all) cases we refer to it as --debug.

`--debug` is also a new flag for subcommand.

I don't think any user would understand why one flag formatting works in one place but not another.